packages/office-ui-fabric-react/src/components/List/List.test.tsx

```diff
@@ -72,6 +72,16 @@ describe('List', () => {
       expect(listRoot.getDOMNode().getAttribute('role')).toEqual('list');
     });
 
+    it("does not set the root element's role in case of an empty list", done => {
+      const wrapper = mount(<List items={mockData(0)} />);
+
+      wrapper.setProps({ items: mockData(0), onPagesUpdated: (pages: IPage[]) => done() });
+
+      const listRoot = wrapper.find(List);
+
+      expect(listRoot.getDOMNode().getAttribute('role')).toEqual(null);
```

Comment:
Perhaps:

```suggestion
      expect(listRoot.getDOMNode().getAttribute('role')).toBeUndefined();
```

Reply:
I tried this and got a fail on this one.

FAIL  src/components/List/List.test.tsx
  ● List › by default › does not set the root element's role in case of an empty list

    expect(received).toBeUndefined()

    Received: null

Reply:
Instead using `.toBeNull();`